docs/io/fits/index.rst

```diff
@@ -309,6 +309,78 @@ would with a dict::
 
     See also :ref:`sphx_glr_generated_examples_io_modify-fits-header.py`.
 
+.. _structural_keywords:
```

Comment:
I think this should go in another page, not in the main one which is already very long. Probably in http://docs.astropy.org/en/latest/io/fits/usage/headers.html#header-of-an-hdu ? As a subsection of "Header of an HDU" to emphasize the close relation between the header and its HDU ?
Then this section could be linked from the main page.

Reply:
I can see both sides of the argument here. This idea originally came from #3836, which, in my mind, was primarily concerned with getting this information in front of new users as quickly as possible. This is why I initially put this on the "Getting Started" page.

On the other hand, I think we're already getting close to "too detailed" for an introduction, so maybe it does make sense to really go into these details in a separate page.

So I guess I'm arguing for at least some mention of this to remain on the "Getting Started" page, but I'm fine adding more information somewhere else. The question then becomes what the minimal useful amount of information for the main page is. Maybe something as simple as a list of all structural keywords without any `*HDU` context and a warning not to touch them?